automation/build-artifacts.repos.el7

```diff
@@ -1,2 +1,3 @@
 http://jenkins.ovirt.org/job/ovirt-engine-nodejs_master_build-artifacts-el7-x86_64/lastSuccessfulBuild/artifact/exported-artifacts/
 http://jenkins.ovirt.org/job/ovirt-engine-nodejs-modules_master_build-artifacts-el7-x86_64/lastSuccessfulBuild/artifact/exported-artifacts/
+http://jenkins.ovirt.org/job/ovirt-engine-yarn_master_build-artifacts-el7-x86_64/lastSuccessfulBuild/artifact/exported-artifacts/
```

Comment:
We can, but perhaps not without a considerable cost in job run time.  (This may also break some ugly hacks we had to make in some places, that are assuming the content of the `*.packages` file is simply appended as args to `yum install`. But that is a lesser concern).

We've kinda been avoiding intrusive changes to `mock_runner.sh` because we want to move forward to something else. But this may indeed be critical enough to address right now.